src/others/average.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Returns the average of a list
+ *
+ * @public
+ *
+ * @example
+ * var average = require('path-to-algorithms/src/others/average').average;
+ * var list = [477, 360, 869, 334, 541]
+ *
+ * console.log(average(list)); // 516.2
+ *
+ * @param {array} the list that you want the average of.
+ *
+ * @module others/average
+*/
+(function (exports) {
+  'use strict';
+
+  function average(array) {
+    const summed = this.sum(array);
```

Comment:
Where's `sum` defined?

Reply:
I copied from my library DisplayJS, and I forgot to add this 😊, will fix